src/main/java/com/amazonaws/services/sqs/AmazonSQSIdleQueueDeletingClient.java

```diff
@@ -210,8 +219,9 @@ public CreateQueueResult createQueue(CreateQueueRequest request) {
         QueueMetadata metadata = new QueueMetadata(queueName, queueUrl, createdAttributes);
         queues.put(queueUrl, metadata);
 
-        metadata.heartbeater = executor.scheduleAtFixedRate(() -> heartbeatToQueue(queueUrl), 
-                0, heartbeatIntervalSeconds, TimeUnit.SECONDS);
+        String finalQueueUrl = queueUrl;
+        metadata.heartbeater = executor.scheduleAtFixedRate(() -> heartbeatToQueue(finalQueueUrl),
+                (long) (Math.random() * 5), heartbeatIntervalSeconds, TimeUnit.SECONDS);
```

Comment:
Consider scaling this to the heartbeat interval period, just as the random sweeping delay is scaled to the sweeping period: https://github.com/awslabs/amazon-sqs-java-temporary-queues-client/blob/f22cf6dcd72e06ebf6493c3264377e84e7ebb31e/src/main/java/com/amazonaws/services/sqs/IdleQueueSweeper.java#L42

Reply:
Done